core/services.go

```diff
@@ -180,6 +180,158 @@ func (app *Application) createGroup(clientID string, current *model.User, group
 	return groupID, nil
 }
 
+func (app *Application) createGroupV3(clientID string, current *model.User, group *model.Group, membershipStatuses model.MembershipStatuses) (*string, *utils.GroupError) {
+
+	var groupError *utils.GroupError
+	var groupID *string
+	err := app.storage.PerformTransaction(func(context storage.TransactionContext) error {
+		var err error
+
+		// Create intitial members if need
+		var members []model.GroupMembership
+		accountIDs := []string{}
```

Comment:
Hi @mdryankov , it looks like we did not handle adding the user that made the call to create the group into the list of members. I think this should be handled on the back-end to ensure that the calling user is always present in this list.

To handle this, I think we can just modify these lines to:

```
accountIDs := []string{current.ID}
		accountIDMapping := map[string]model.MembershipStatus{
			current.ID: {
				UserID: current.ID,
				Status: "admin",
			},
		}
```